api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -231,11 +238,26 @@ private User initializeUserIfNeeded() {
     // not accepting the terms of use.
 
     User user = initializeUserIfNeeded();
+    String blockscoreStatus = "not-verified";
     try {
-      return ResponseEntity.ok(profileService.getProfile(user));
+      String blockscoreId = user.getBlockscoreId();
+      if (blockscoreId != null) {
+        Person person = blockscoreService.retrievePerson(blockscoreId);
+        Field statusField = Person.class.getDeclaredField("status");
+        statusField.setAccessible(true);
+        blockscoreStatus = statusField.get(person).toString();
+      }
+      return ResponseEntity.ok(
+        profileService.getProfile(user)
+          .idVerificationStatus(blockscoreStatus)
```

Comment:
Does the verification status from BlockScore get set to something immediately and not change? Or do we have to poll for updates?

Reply:
Ditto.